backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc

```diff
@@ -16,6 +16,9 @@ include::modules/dr-restoring-cluster-state.adoc[leveloffset=+1]
 include::modules/dr-scenario-cluster-state-issues.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-.Additional resources
+[id="additional-resources"]
+== Additional resources
 
-* See xref:../../../networking/accessing-hosts.adoc#accessing-hosts[Accessing the hosts] for how to create a bastion host to access {product-title} instances and the control plane nodes with SSH.
+* xref:../../../networking/accessing-hosts.adoc#accessing-hosts[Creating a bastion host to access {product-title} instances and the control plane nodes with SSH].
```

Comment:
In my view, these two bulleted items are still not parallel. You could either make both of them just the heading from the linked resource, with no full stop at the end, or you could make them full sentences with full stops at the end. My preference would be to use the headings without full stops, but I'll leave that up to you.

At the moment, one of them has a full stop and a full description and the other is just the heading with no full stop. I think it would be good to make them consistent.